examples/BlazorServerApp/README.md

```diff
@@ -0,0 +1,69 @@
+# Blazor Server App
+
+This sample shows the recommended way to use the Feature Management library in a Blazor Server App.
+
+## Quickstart
+
+To run this sample, follow these steps:
+
+1. Set the project as the startup project.
+2. Run the project.
+3. Refresh the page until the BETA content occurs.
+
+## About the App
+
+This app demonstrates how to use the Feature Management library in Blazor apps.
+
+This app uses two feature flags: "BrowserEnhancement" and "Beta".
+
+``` json
+"FeatureManagement": {
+    "BrowserEnhancement": {
+        "EnabledFor": [
+            {
+                "Name": "Browser",
+                "Parameters": {
+                    "AllowedBrowsers": [ "Edge" ]
```

Comment:
the implementation of Browser filter doesn't accept allowed browsers. Although, I expect it should.

Reply:
Oh, I used to use parameters to specify the allowed browser, just like the BrowserFilter in the FeatureFlagDemo example. Then I removed this parameters to keep it simple. I can change it back to use parameters.